tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness/RunProcessor.cs

```diff
@@ -60,6 +60,41 @@ private Guid GetCollectionIdFromProject(TeamProject project)
             return collectionId;
         }
 
+        private async Task<VssConnection> GetVssConnectionAsync(string organization)
+        {
+            var vssConnectionCacheKey = $"{organization}_vssConnectionCacheKey";
+
+            var connection = await cache.GetOrCreateAsync(vssConnectionCacheKey, async (entry) =>
+            {
+                var azureDevOpsPersonalAccessToken = await GetAzureDevOpsPersonalAccessTokenAsync();
+                var credentials = new VssBasicCredential("nobody", azureDevOpsPersonalAccessToken);
```

Comment:
Should "nobody" be a constant somewhere?

Reply:
Its never going to need to change, username is ignored by Azure DevOps for PAT-based auth.